src/measure.cpp

```diff
@@ -199,4 +199,5 @@ double measure( moab::EntityType type,
       return 0.0;
   }
 }
-      
+
+
```

Comment:
Why this extra line?

Reply:
an accident.

nice catch tho !

Reply:
It was lucky...  When I review these whitespace PR's, I set Github to ignore whitespace and then I should see no other changes.  Although I would consider this to also be whitespace, it showed it anyway